querier.hh

```diff
@@ -141,8 +154,9 @@ public:
             dht::partition_range range,
             query::partition_slice slice,
             const io_priority_class& pc,
-            tracing::trace_state_ptr trace_ptr)
-        : querier_base(schema, permit, std::move(range), std::move(slice), ms, pc, std::move(trace_ptr))
+            tracing::trace_state_ptr trace_ptr,
+            std::optional<querier_config> config)
```

Comment:
You can replace with `querier_config config = {}` here instead to reduce the churn.

Reply:
ok, i also add default constructor for querier_config.